privacyidea/lib/smsprovider/FirebaseProvider.py

```diff
@@ -127,7 +127,31 @@ def submit_message(self, firebase_token, data):
         fcm_message = {
             "message": {
                         "data": data,
-                        "token": firebase_token
+                        "token": firebase_token,
+                        "android": {
+                                    "priority": "HIGH",
+                                    "ttl": "120s",
+                                    "fcm_options": {"analytics_label": "AndroidPushToken"}
+                                   },
+                        "apns": {
+                                 "headers": {
+                                             "apns-priority": "10",
+                                             "apns-push-type": "alert",
+                                             "apns-collapse-id": "privacyidea.pushtoken",
+                                             "apns-expiration": str(int(time.time()) + 120)
```

Comment:
@NuvandaPV Would is make sense to make such values configurable?

Reply:
In theory it would make sense to make this configurable, however I am not sure whether that would be worth the effort to add extra configuration options, just so the administrator can pick what ringtone to use on iOS and whether the notification should persist at the top of the screen, or hide after a couple seconds.

Reply:
I am not thinking that much about different ring tones than future changes in the API, where new key/values might be necessary for push to run.
So maybe we initialize the complete fcm_message from a config file that ships as default with privacyIDEA and can be changed if necessary. (Maybe this can also be an issue for version 3.3)?

Reply:
You mean changes in Firebase's API?

Reply:
Yes. Changes. Enhancemets. Things we missed. Just like now.